### PR TITLE
fix(seed): typo in Responsive Design with Bootstrap

### PR DESCRIPTION
**Pull request by build65**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [ ] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [ ] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [ ] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [ ] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #XXXXX

#### Description

Fixed the typo at line 45 from 

rel=\"stylesheet\" href=\"//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css\

TO:

href=\"https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css\

added https so it works when you copy and paste it in an external text editor.

### CODE REVIEW

**Comment by build65** [comment]
I was going through lessons and when I wanted to build this challenge using my text editor, Bootstrap declaration didn't work because it needs "https". Hope this little change helps learners have smoother experience.

File: .gitignore [hunk 1]
```diff
@@ -56,3 +56,4 @@ public/css/main*
 webpack-bundle-stats.html
 server/rev-manifest.json
 google-credentials.json
+/.vs
```
Comment (raisedadead) [inline .gitignore]: What is this about?

**Comment by raisedadead** [comment]
On that note, this is not a typo and was actually intentional originally.
See https://www.paulirish.com/2010/the-protocol-relative-url/

But, I believe, this something we should be NOT be doing now on, since the author have long since updated their advice.

More reference: https://jeremywagner.me/blog/stop-using-the-protocol-relative-url/

Thanks for the PR, please clarify the review comment in a previous comment, and we are good to go?

File: .gitignore (hunk 1 above)
Comment (build65) [reply .gitignore]: Visual Studio uses a hidden folder called .vs to store personal settings and preferences. I ignored it so it is not included in my push. I can push again using git bash if this is making issues.

**Comment by raisedadead** [comment]
Hi @build65 looks like you have deleted the `.gitignore` completely?

**Pull request closed without merging**